include/NovelRT.h

```diff
@@ -58,9 +65,8 @@ namespace NovelRT {
 }
 
 namespace NovelRT::Audio {
-  typedef std::map<std::string, Mix_Chunk*> SoundBank;
-  typedef std::map<std::string, Mix_Music*> MusicBank;
-  typedef std::map<std::string, int> ChannelMap;
+  typedef std::vector<ALuint> SoundBank;
```

Comment:
Just note for the audio branch...

This should become an explicit type, that way we aren't tied to `std::vector` nor to `ALuint`